clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -1277,10 +1279,12 @@ private void releaseAssignmentAndLeaveGroup(final Timer timer) {
         UnsubscribeEvent unsubscribeEvent = new UnsubscribeEvent(calculateDeadlineMs(timer));
         applicationEventHandler.add(unsubscribeEvent);
         try {
-            // If users subscribe to an invalid topic name, they will get InvalidTopicException in error events,
-            // because network thread keeps trying to send MetadataRequest in the background.
-            // Ignore it to avoid unsubscribe failed.
-            processBackgroundEvents(unsubscribeEvent.future(), timer, e -> e instanceof InvalidTopicException);
+            // If users subscribe to a topic with invalid name or without permission, they will get some exceptions.
+            // Because network thread sends MetadataRequest and FindCoordinatorRequest in the background,
```

Comment:
I think emphasis the background thread `keep trying to send MetadataRequest and FindCoordinatorRequest` is better, WDTY?

Reply:
Updated it. Thanks.